kremling-inline-loader.md

```diff
@@ -0,0 +1,105 @@
+# Kremling inline loader
+
+The `kremling-inline-loader` is a webpack loader that processes your
+javascript that have`k` tagged templates in them. You can learn more about the
+`k` tagged template and how to use it in the
+[Kremling docs](https://kremling.js.org/api/k-tagged-template.html).
+
+Just like the [Kremling loader](https://kremling.js.org/walkthrough/kremling-loader.html),
+the `kremling-inline-loader` processes `css` using `postcss` so you don't have
+to use ampersands. Unlike the `kremling-loader` though, you can do all of this
+inside your js!
+
+### How it works
+
+The `kremling-inline-loader` parses `js` files during build time and looks for
+`k` tagged template literals. eg:
+
+```javascript
+const styles = k`
+  .test {
+    background-color: red;
+  }
+`;
+```
+
+Normally, kremling would require ampersands (`&`) before each rule so it knew
+where to add the scoping. But with the power of `postcss`, we can omit the
+ampersands and add the scoping attributes to all the styling rules.
+
+### Install
+
+```shell
+$ npm install kremling-loader -D
```

Comment:
First off -- see my other comment about how this might be better as a babel plugin instead of a webpack loader.

If we do end up going with a webpack loader, though:

The kremling-loader is now doing two pretty different things. Should we make kremling-inline-loader its own npm package? So that it is easier for people to find a github repo and documentation for it, and so they don't have to reach into the src directory of the kremling-loader node modules to use it.

Alternatively, maybe even better would be if the kremling-loader was smart enough to handle both situations without any extra configuration. Meaning that if it processes a css file, it just parses the whole file, whereas if it is processing a js file, it looks for the `k` tagged template literals.

The webpack config would look like this:
```js
module.exports = {
  module: {
    rules: [
      test: /\.(js|css)$/,
      use: [
        { loader: 'kremling-loader' }
      ]
    ]
  }
}
```